plugins.d/alarm-notify.sh

```diff
@@ -1339,6 +1363,62 @@ send_telegram() {
 }
 
 # -----------------------------------------------------------------------------
+# Microsoft Team sender
+
+send_msteam() {
+
+    local webhook="${1}" channels="${2}" httpcode sent=0 channel color payload
+
+    [ "${SEND_MSTEAM}" != "YES" ] && return 1
+
+    case "${status}" in
+        WARNING)  icon="${MSTEAM_ICON_WARNING}" && color="${MSTEAM_COLOR_WARNING}";;
+        CRITICAL) icon="${MSTEAM_ICON_CRITICAL}" && color="${MSTEAM_COLOR_CRITICAL}";;
+        CLEAR)    icon="${MSTEAM_ICON_CLEAR}" && color="${MSTEAM_COLOR_CLEAR}";;
+        *)        icon="${MSTEAM_ICON_DEFAULT}" && color="${MSTEAM_COLOR_DEFAULT}";;
+    esac
+
+    for channels in ${channels}
+    do
+        ## More details are available here regarding the payload syntax options : https://docs.microsoft.com/en-us/outlook/actionable-messages/message-card-reference
+        ## Online designer : https://acdesignerbeta.azurewebsites.net/
+        payload="$(cat <<EOF
+        {
+            "@context": "http://schema.org/extensions",
+            "@type": "MessageCard",
+            "themeColor": "${color}",
+            "title": "$icon Alert ${status} from netdata for ${host}",
+            "text": "${host} ${status_message}, ${chart} (_${family}_), *${alarm}*",
+            "potentialAction": [
+            {
+                "@type": "OpenUri",
+                "name": "Netdata",
+                "targets": [
+                { "os": "default", "uri": "http://$(hostname -f):19999" }
```

Comment:
please do not assume the URL is like that. A lot of people proxy netdata via other web server or use VPNs that prevent dns resolution. Also, this will resolve to the master netdata when it processes alarms for a slave.

You can use `${goto_url}` that uses the registry and the netdata unique id, to divert to the proper server.